@xen-orchestra/lite/src/components/UsageBar.vue

```diff
@@ -3,25 +3,31 @@
     <div class="header">
       <slot name="header" />
     </div>
-    <template v-if="data !== undefined">
-      <ProgressBar
+    <div v-if="data !== undefined">
+      <div
         v-for="item in computedData.sortedArray"
         :key="item.id"
-        :value="item.value"
-        :label="item.label"
-        :badge-label="item.badgeLabel"
-      />
+        class="progress-item"
+      >
+        <UiProgressBar :value="item.value" color="custom" />
+        <div class="badge">
+          <span class="circle" />
+          {{ item.label }}
+          <UiBadge v-if="item.badgeLabel">{{ item.badgeLabel }}</UiBadge>
```

Comment:
IMO we need to keep the previous behavior. If there is no `badgeLabel`, the percentage is displayed.
```suggestion
          <UiBadge>{{ item.badgeLabel ?? `${item.value}%` }}</UiBadge>
```

Reply:
Good point! You are totally right.